code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -684,6 +684,13 @@ wink, yawn, swish, sway/wag, fastsway/qwag, stopsway/swag, beep, ping, buzz, sla
 
 	pose =  sanitize(input(usr, "This is [src]. [get_visible_gender() == MALE ? "He" : get_visible_gender() == FEMALE ? "She" : "They"]...", "Pose", null)  as text)
 
+/mob/living/carbon/human/verb/feel()

```

Comment:
who'll even use this lmao

Reply:
me

Reply:
It might be a good idea to let them detect other shit? Like if they're interacting with anything with a do_after or something like that. a niche-case verb seems meh at best

Reply:
The proc name is unintuitive.
Use set_thoughts() or something simmilar that actually describes what it does